apps/api/src/app/events/services/workflow.queue.service.ts

```diff
@@ -12,6 +12,7 @@ export class WorkflowQueueService {
       db: Number(process.env.REDIS_DB_INDEX),
       port: Number(process.env.REDIS_PORT),
       host: process.env.REDIS_HOST,
+      password: process.env.REDIS_PASSWORD,
```

Comment:
_For the reader's convenience..._

BullMQ's `QueueBaseOptions` (https://github.com/taskforcesh/bullmq/blob/89bad097edf1692dba19eec938da82b6f2be7430/src/interfaces/redis-options.ts)
is extending `IORedis.RedisOptions` that is defined here:
(https://github.com/luin/ioredis/blob/eabb70ed835828ad85b602a217992486e05d2218/lib/cluster/util.ts)